### DISCUSSION TITLE
Unconfirm Wallet Balance

### DISCUSSION THREAD

**Discussion by britvr** [lightningnetwork/lnd#8668] [Q&A]
Hey guy,

I'm trying to get my Bitcoin off the LND, but I'm running into a problem. The Bitcoin is unconfirmed. 
![image](https://github.com/lightningnetwork/lnd/assets/167540638/d084475a-592c-4373-a1a4-9e44f61a5581)

**Comment by hieblmi** [comment]
Do you have pending channel closures? `lncli pendingchannels`

**Reply by britvr** [reply]
I imported the command to lncli closeallchannels last night and today. 
![image](https://github.com/lightningnetwork/lnd/assets/167540638/9d99f954-3313-4f15-8f09-abf420e48d6e)

**Reply by britvr** [reply]
![image](https://github.com/lightningnetwork/lnd/assets/167540638/4b0cb7cc-bcf3-4ba1-8957-40af0a3f5d98)
It saying I have no pending channels, but I have inactive channels.

**Comment by hieblmi** [comment]
> I imported the command to lncli closeallchannels last night and today. 

Did you execute `lncli closeallchannels`? From the screenshot I can see that there are elements in the `pending_force_closing_channels` array, I think this is where you find your unconfirmed balance?

**Reply by britvr** [reply]
The unconfirm changed over, but during that process, it looks like I had a fee 1million sats. 🤷🏾‍♂️🤷🏾‍♂️

**Reply by hieblmi** [reply]
Since I can't see your transactions I can only assume that force closures during the recent mempool fee surge caused that high of a transaction fee. Are there any things unclear to you at this point?

**Reply by britvr** [reply]
Yes, it has been over a week, and I still have 10 inactive channels. I request to closeallchannels and only 4 channels closed. Do I have to closeallchannel --force again? 
![image](https://github.com/lightningnetwork/lnd/assets/167540638/63fbdcc1-e3ac-44ed-8050-a923b6d41c74)

**Reply by hieblmi** [reply]
Inactive channels won't close the channel cooperatively. That's why the `closeallchannels` didn't act on those inactive channels. I would try to see if your channel peers come online and your channels become active. Then trying closing them all again(if that's what you want to do). However the current mempool fees are pretty high so you might want to wait till fees come down.

**Reply by britvr** [reply]
How long does a --force all channels need to close? Because I forced all channels last week, and only 4 close.

**Reply by hieblmi** [reply]
If you look at the pending channel closures you should find their linked transactions in the mempool. Once a transaction confirms you have to wait some time, depending on the channel size. Due to the current mempool fees this might take a while.

**Comment by hieblmi** [comment]
Closing this as the force-closure behavior is to be expected. Please re-open if things are still unclear.
